modules/PropTypes.js

```diff
@@ -1,48 +1,50 @@
 import { PropTypes } from 'react'
+const {func, string, any, oneOf, shape} = PropTypes
```

Comment:
Nit: Add spaces inside of the brackets. That's consistent with how we normally write destructurings.